src/Kiota.Builder/CodeDOM/CodeMethod.cs

```diff
@@ -124,9 +125,11 @@ public static CodeMethod FromIndexer(CodeIndexer originalIndexer, Func<string, s
         get; set;
     }
     public string RequestBodyContentType { get; set; } = string.Empty;
-#pragma warning disable CA2227
-    public HashSet<string> AcceptedResponseTypes { get; set; } = new(StringComparer.OrdinalIgnoreCase);
-#pragma warning restore CA2227
+#pragma warning disable CA1002
+    public List<string> AcceptedResponseTypes { get; private set; } = new();
```

Comment:
Using:
```
public IList<string> AcceptedResponseTypes { get; private set; } = new List<string>();
```
should get rid of the `CA1002` warning according to [this](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1002)

Reply:
updated